packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -175,6 +183,9 @@ const registerCommands = (): vscode.Disposable => {
     'sf.launch.apex.replay.debugger.with.current.file',
     launchApexReplayDebuggerWithCurrentFile
   );
+  const notifyA4DExtensionActivated = vscode.commands.registerCommand('sf.notify.a4d.extension.activated', () =>
```

Comment:
@mingxuanzhangsfdx I'd rather not add a new command to a4d to support this.

Maybe we can discuss approaches today.